Makefile

```diff
@@ -403,20 +404,23 @@ generate-go-conversions-capd: $(CONVERSION_GEN) ## Generate conversions go code
 		--build-tag=ignore_autogenerated_capd \
 		--extra-peer-dirs=sigs.k8s.io/cluster-api/api/v1alpha3 \
 		--extra-peer-dirs=sigs.k8s.io/cluster-api/api/v1alpha4 \
-		--output-file-base=zz_generated.conversion $(CONVERSION_GEN_OUTPUT_BASE) \
+		--output-file-base=zz_generated.conversion $(CONVERSION_GEN_OUTPUT_BASE_CAPD) \
 		--go-header-file=../../../hack/boilerplate/boilerplate.generatego.txt
 
+# The tmp/sigs.k8s.io/cluster-api symlink is a workaround to make this target run outside of GOPATH
 .PHONY: generate-go-openapi
 generate-go-openapi: $(OPENAPI_GEN) $(CONTROLLER_GEN) ## Generate openapi go code for runtime SDK
-	@for pkg in "api/v1beta1" "$(EXP_DIR)/runtime/hooks/api/v1alpha1"; do \
+	@mkdir -p ./tmp/sigs.k8s.io; ln -s $(ROOT_DIR) ./tmp/sigs.k8s.io/; cd ./tmp; \
+	for pkg in "api/v1beta1" "$(EXP_DIR)/runtime/hooks/api/v1alpha1"; do \
 		$(MAKE) clean-generated-openapi-definitions SRC_DIRS="./$${pkg}"; \
 		echo "** Generating openapi schema for types in ./$${pkg} **"; \
 		$(OPENAPI_GEN) \
-			--input-dirs=./$${pkg} \
+			--input-dirs=sigs.k8s.io/cluster-api/$${pkg} \
 			--output-file-base=zz_generated.openapi \
 			--output-package=sigs.k8s.io/cluster-api/$${pkg} \
-			--go-header-file=./hack/boilerplate/boilerplate.generatego.txt; \
-	done
+			--go-header-file=../hack/boilerplate/boilerplate.generatego.txt; \
+	done; \
+	rm sigs.k8s.io/cluster-api
```

Comment:
probably has to match ./tmp/sigs.k8s.io above

Reply:
I'm `cd`ing into that directory, so no

Reply:
Ups, you're right